core/src/main/java/com/hcl/ecomm/core/servlets/CreateOrderServlet.java

```diff
@@ -0,0 +1,108 @@
+package com.hcl.ecomm.core.servlets;
+
+import com.hcl.ecomm.core.services.CreateOrderService;
+import com.hcl.ecomm.core.services.DeleteCartItemService;
+import org.apache.commons.httpclient.HttpStatus;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.SlingHttpServletResponse;
+import org.apache.sling.api.servlets.HttpConstants;
+import org.apache.sling.api.servlets.SlingAllMethodsServlet;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.osgi.framework.Constants;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.Servlet;
+import javax.servlet.ServletException;
+import java.io.BufferedReader;
+import java.io.IOException;
+
+/**
+ * Servlet that delete the existing cart item by hitting the Magento service.
+ */
```

Comment:
@nitheshkumar-hcl  update the servlet class description

Reply:
updated the  servlet class description